content/doc/book/pipeline/syntax.adoc

```diff
@@ -274,7 +286,21 @@ dockerfile:: Execute the Pipeline, or stage, with a container built from a
 
 ////
 
+===== Additional Options
+
+These are a few options for two or more `agent` implementations. They are not
```

Comment:
I'm basically working under the assumption that `label` is freakin' weird and just backing away slowly from it. =)